tsstats/templates/index.jinja2

```diff
@@ -49,5 +59,23 @@
   {% endfor %}
   <small>Generated by <a href="https://github.com/Thor77/TeamspeakStats" rel="noopener">TeamspeakStats</a> at {{ creation_time|frmttime }}</small>
 </div>
+
+<script type="text/javascript">
+  document.addEventListener('click', function(event) {
+    var toggler = event.target;
+    if (toggler.dataset.toggle !== 'collapse') {
+      return;
+    }
+    var toggleTarget = document.getElementById(toggler.dataset.target);
+
+    if (toggleTarget.classList.contains('collapse')) {
+      toggleTarget.classList.remove('collapse');
```

Comment:
I would prefer to just rely on `aria-expanded`, because adding/removing classes just for saving a state seems kinda wrong to me.